src/main/proto/cosmas.proto

```diff
@@ -37,9 +37,21 @@ message CreateVersionRequest {
 message CreateVersionResponse {
 }
 
+message CreatePatchRequest {
+    string user = 1;

```

Comment:
I suggest naming it `userId`. Because it is user id indeed and because the next field is called fileId (which tells that we are using "id" suffix to indicate id)